exec/java-exec/src/test/java/org/apache/drill/exec/physical/impl/scan/v3/ScanFixture.java

```diff
@@ -81,8 +81,8 @@ public ScanFixture build() {
       Scan scanConfig = new AbstractSubScan("bob") {
 
```

Comment:
Bob?

Reply:
It was added in the scope of from DRILL-7701, so I'm not sure about the intention, perhaps some random name that may be also used in some tests.
Or this name is used intentionally if we also have Alice somewhere in the code 🙂